NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs

```diff
@@ -158,33 +158,31 @@ public async Task StartAsync(bool embedMetadata)
         _imgStatus.SetFromIconName("folder-download-symbolic");
         _viewStackState.SetVisibleChildName("downloading");
         _progLabel.SetText(_localizer["DownloadState", "Preparing"]);
-        var success = await _download.RunAsync(embedMetadata, new Progress<DownloadProgress>((x) =>
+        _lblFilename.SetText(_download.Filename);
```

Comment:
```suggestion
```

No longer needed, was a remnant from before we fetched the video title for the default file name before creating the row